src/stravalib/field_conversions.py

```diff
@@ -55,14 +54,6 @@ def enum_values(enums: Sequence[ActivityType | SportType]) -> list[str | None]:
     return [enum_value(e) for e in enums]
 
 
-@optional_input
-def time_interval(seconds: int) -> timedelta:
```

Comment:
i have two questions

1. can you help me understand what this function's purpose was before - i understand it is a legacy behavior function BUT, all it did was return an timedelta object. 
2. If we remove it altogether - are we breaking something for some 
i see it has an optional decorator but i'm still wrapping my head around what removing it means for users (and thinking about what it will look like to migrate to pydantic 2.0 and perhaps drop some of this legacy behavior support

Reply:
perhaps it is true that no one would call this directly but it is not defined as a private method.

Reply:
1. The only thing it is doing is turning seconds into a timedelta it was probably needed to pass to the field conversion. Because of the decorator, it is actually doing a bit more. If you pass it None it returns None
2. In theory it could break someone's if they imported and used this function directly. It's really unlikely to be the case I think.